### PR TITLE
chore: Jest configuration is now decoupled from package.json and can be easily customized and maintainable.

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝
Decoupled jest configurations from package.json.
This improves maintainability of jest configurations.

## How to test 🧪

- Validate running unit test
- Validate building the app and packages.
- Validate running cypress tests

**How do I run relevant unit or e2e tests?**
yarn test
yarn cy:run

### CODE REVIEW

File: packages/manager/package.json [hunk 1]
```diff
@@ -212,96 +212,7 @@
     "vite-plugin-svgr": "^2.4.0"
   },
   "jest": {
```
Comment (bnussman-akamai) [inline packages/manager/package.json:214]: I think we can completely remove the `jest` section from the `package.json`.

**Review by bnussman-akamai — COMMENTED** [review]
Nice!

Did using `jest.config.ts` not work? Our jest version might be too old to understand a `.ts` config file

**Comment by cpathipa** [comment]
> Nice!
> 
> Did using `jest.config.ts` not work? Our jest version might be too old to understand a `.ts` config file

I think .ts may need ts-node package to run the tests. May be that's due to older version.

**Comment by jaalah-akamai** [comment]
> I think .ts may need ts-node package to run the tests. May be that's due to older version.

Yea we'll need `ts-node` to run TypeScript files directly. You will also need `ts-jest` for this as well.

```
import type { Config } from '@jest/types';

const config: Config.InitialOptions = {
 // Configs...

**Pull request closed without merging**